ci/upstream.yml

```diff
@@ -1,86 +1,73 @@
----
```

Comment:
These changes look superfluous...

Reply:
the drop of the `---` or all the changes in the file?  If the latter, it is because the file is now autogenerated by the `ci/scripts/build-upstream-list` script

Reply:
That's horrifying.